packages/bind/spec/crossWindowBehaviors.ts

```diff
@@ -66,11 +65,11 @@ describe('Cross-window support', function () {
 
     win2.document.write(BLANK_HTML)
     win2.document.close()
-    body2 = win2.document.body
+    const body2 = win2.document.body
```

Comment:
👍

Reply:
With eslint we can later fix all detections to let or const.. we are all already prepared, see https://github.com/Auge19/tko/tree/add_eslint

Reply:
Incidentally I'd be ok with biome too, though I think eslint is a more mature and stable.